control/veda_deliver.py

```diff
@@ -228,13 +222,22 @@ def _INFORM_INTAKE(self):
             self.auth_dict['veda_deliverable_bucket'],
             self.encoded_file
         ))
-        os.system(
-            ' '.join((
-                'wget -O',
-                os.path.join(self.node_work_directory, self.encoded_file),
-                self.hotstore_url
-            ))
+
+        conn = S3Connection()
+        try:
+            bucket = conn.get_bucket(self.auth_dict['veda_deliverable_bucket'])
+        except S3ResponseError:
+            logger.error('[VIDEO_DELIVER] Invalid Storage Bucket')
+            return
+        source_key = bucket.get_key(self.encoded_file)
+        if source_key is None:
+            logger.error('[VIDEO_DELIVER] S3 Intake Object NOT FOUND')
+            return
+
+        source_key.get_contents_to_filename(
+            os.path.join(self.node_work_directory, self.encoded_file)
```

Comment:
Why do we need this change, since we're not using the output anywhere ?

Reply:
This is the file to reupload to the final endpoint (final s3 bucket, youtube), it has to be downloaded to the worker first. Once we deprecate youtube we can probably refactor this to a simple aws s3 copy command or eliminate the need for this worker to handle files at all, as the encode workers can upload the final encoded files to the endpoint bucket.
This was using wget previously, but the new IAM worker roles are incompatible with it. The stale worker hid this bug previously.

Reply:
We should create ticket for this, this should be done with youtube  deprecate work ?